com.unity.multiplayer.mlapi/Runtime/Core/RpcBatcher.cs

```diff
@@ -89,10 +89,13 @@ private static void FillTargetList(in RpcFrameQueueItem queueItem, ref ulong[] n
         {
             switch (queueItem.queueItemType)
             {
+                // todo: revisit .resize() and .ToArry() usage, for performance
                 case RpcQueueContainer.QueueItemType.ServerRpc:
                     Array.Resize(ref networkIdList, 1);
                     networkIdList[0] = queueItem.networkId;
                     break;
+                default:
```

Comment:
if you also considered and tested changes for `RpcQueueContainer.QueueItemType.CreateObject` and `RpcQueueContainer.QueueItemType.DestroyObject`?

Reply:
I hear that 
> In RpcProcessingQueue that should be completely isolated.  The batcher only receives RPCs...
which would explain why this was not a problem so far. @NoelStephensUnity 

I'm still of the opinion that the `default:` there is an improvement.

Reply:
sure, that's alright then